### PR TITLE
Add Sentry

### PR DESCRIPTION
**Pull request by YtvwlD**
This PR adds Sentry as an application. I chose Sentry 20.06 because it's the one we're currently using on its own VM.

I have tested this in so far that the application can be configured, it boots and I'm able to log in to the Web UI and click around.
I couldn't test importing our previous data because I just don't have the hardware for it.

And this might be the biggest problem: Sentry seems to explode with each release (this PR adds 18 containers) and consumes quite a bit of resources even though it should do next to nothing. I'm not even sure it will run on our hardware.

Perhaps we're better off using [GlitchTip](https://glitchtip.com/)? Sadly, I didn't see an option to import data from Sentry.

### CODE REVIEW

File: enabled/sentry.yml [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+  # symbolicator-cleanup:
+  #   << : *defaults
+  #   image: symbolicator-cleanup-onpremise-local
+  #   build:
+  #     context: ./cron
+  #     args:
+  #       BASE_IMAGE: 'getsentry/symbolicator:$SYMBOLICATOR_VERSION'
+  #   command: '"55 23 * * * gosu symbolicator symbolicator cleanup"'
+  #   volumes:
+  #     - 'symbolicator:/data'
```
Comment (nomaster) [inline enabled/sentry.yml:175]: what's with this commented out stuff?

File: enabled/sentry.yml [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+  #   << : *defaults
+  #   image: 'getsentry/relay:20.6.0'
+  #   volumes:
+  #     - type: bind
+  #       read_only: true
+  #       source: ./relay
+  #       target: /work/.relay
+  #   depends_on:
+  #     - kafka
+  #     - redis
```
Comment (nomaster) [inline enabled/sentry.yml:236]: more commented out stuff

File: enabled/sentry.yml [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+volumes:
+  data:
+  postgres:
+  redis:
+  zookeeper:
+  kafka:
+  clickhouse:
+  # symbolicator:
```
Comment (nomaster) [inline enabled/sentry.yml:244]: and a little more

**Review by nomaster — CHANGES_REQUESTED** [review]
Nice! I just have questions about the commented out stuff in the docker-compose file. Maybe you want a separate file for services that are only activated on demand?

Regarding the resources, can you say how much memory we need for the deployment and how much more that is compared to before?

**Comment by YtvwlD** [comment]
The commented out stuff are optional services that we don't currently need (at least, I think so): mainly parsers for core dumps.

The software boots with 2,5GB RAM on my laptop, but that's not enough to import our old data.

**Pull request closed without merging**